script/prerequisitesForGetToolchains.sh

```diff
@@ -3,6 +3,11 @@
 # This shell script add PPA to get onecc-docker package for ONEToolchain.
 
 PPA_NAME="ppa:one-compiler/onecc-docker"
-
 add-apt-repository -y ${PPA_NAME}
+
+# This shell script add PPA to get edgetpu-compiler package for EdgeTPUToolchain.
+
+curl https://packages.cloud.google.com/apt/doc/apt-key.gpg | sudo apt-key add -
+echo "deb https://packages.cloud.google.com/apt coral-edgetpu-stable main" | sudo tee /etc/apt/sources.list.d/coral-edgetpu.list
+
 apt-get update -yqq
```

Comment:
FYI, this part is better be seperated from the current script. Let's distinguish prerequisite stage for onecc and edgetpu.

Reply:
It seems like separating the prerequisites for `onecc` and `edgetpu` would be a good idea.

Is it okay to split `prerequisitesForGetToolchains.sh` into `prerequisitesForGetOneToolchain.sh` and `prerequisitesForGetEdgeTPUToolchain.sh`?

Reply:
Yes, that's cool :_D